app/views/games/show.html.erb

```diff
@@ -8,7 +8,13 @@
 					<td class="<%= color_block(row, col)%>">
 						<% @pieces.each do |piece| %>
 							<% if (piece.row_position == row) && (piece.col_position == col) %>
-								<%= link_to "", "", :class => "#{assign_piece(piece.type, piece.color)}" %>
+
+								<%= link_to edit_piece_path, :method=> 'GET' do %>
```

Comment:
Did you try running this?  You should get an error.  Look at your routes to determine why.